publisher-sdk-tests/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,17 @@
   xmlns:tools="http://schemas.android.com/tools"
   package="com.criteo.publisher.tests">
 
+  <!-- MoPub requires minSdk=19, while the SDK had minSdk=16 -->
```

Comment:
Can you add a `See https://developer.android.com/studio/build/manifest-merge` comment please?

Reply:
done